external/implementations/samlconf-keycloak-impl/build.gradle

```diff
@@ -0,0 +1,14 @@
+group = 'org.codice.samlconf.plugins'
+description = 'Plugin implementation for Keycloak'
```

Comment:
✏️ `Implementation for Keycloak` instead

Reply:
Fixed.